.github/workflows/release.yml

```diff
@@ -203,3 +203,41 @@ jobs:
           for f in $(find dist/ -name '*.tgz'); do
             gh release upload $tag $f
           done
+
+      - name: Add charts to branch
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          version=$(jq -r '.version' <<< "${{ steps.goreleaser.outputs.metadata }}")
+          branch_version=$(cut -d'.' -f1,2 <<< "$version")
+          charts_branch=charts/$branch_version
+
+          if [ ! -e ~/.gitconfig ]; then
+            git config --global user.name "fleet-bot"
+            git config --global user.email fleet@suse.de
+          fi
+
+          echo "publishing helm chart in the branch $charts_branch"
+          if ! git show-ref --quiet "refs/heads/$charts_branch"; then
+            git checkout --orphan "$charts_branch"
+            git rm -rf .
+
+            echo "# Fleet Helm Charts for $branch_version versions" > README.md
+            echo "The documentation is centralized in a unique place, checkout https://fleet.rancher.io/." >> README.md
+
+            git checkout main -- .gitignore
+
+            git add README.md .gitignore
+            git commit -m "Initial commit for $charts_branch"
+          else
+            git checkout "$charts_branch"
+          fi
+
+          mkdir charts
+          find dist/ -name '*.tgz' -exec tar -xf {} -C charts/ \;
```

Comment:
Why do we need to do this, since we've edited values files in-place during step ` Upload charts to release`?
Couldn't we simply reuse the `charts` dir in its current state? Or what am I missing?

Reply:
Oh, I see, thanks for clarifying :+1: